src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -104,4 +108,10 @@ class Repository(
     val visit = visitRepository.findById(visitId!!).orElseThrow()
     visit.visitStatus = visitStatusRepository.findById(VisitStatus.NORM).orElseThrow()
   }
+
+  fun updateCreatedToMatchVisitStart() {
+    val sql =
+      "UPDATE offender_visits SET CREATE_DATETIME = START_TIME"
+    jdbcTemplate!!.execute(sql)
+  }
```

Comment:
the CREATE_DATETIME is a DB system timestamp on creation. Setting the CREATE_DATETIME to match the visit date makes the integration tests testing date ranges clearer